### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by Abel-desu**
Understand the concept of state management.
Use store, actions and reducers in React.
Connect React and Redux.
Connect an existing API via Redux and use the data to display something in a UI.

### CODE REVIEW

File: src/Redux/books/book.js [hunk 1]
```diff
@@ -1,39 +1,68 @@
...
-    return [
-      ...state,
-      {
-        id: action.id,
-        title: action.title,
-        author: action.author,
-      },
-    ];
-  }
+// eslint-disable-next-line
```
Comment (arslanbisharat) [inline src/Redux/books/book.js]: - [ ] Let's not disable linter rules.

File: src/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+/*eslint-disable*/
```
Comment (arslanbisharat) [inline src/components/Form.js]: - [ ] Let's not disable linter rules.

File: src/components/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+/*eslint-disable*/
```
Comment (arslanbisharat) [inline src/components/Navigation.js]: - [ ] Let's not disable linter rules.

File: src/components/book-item.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
+/*eslint-disable*/
```
Comment (arslanbisharat) [inline src/components/book-item.js]: - [ ] Let's not disable linter rules.

File: src/index.js [hunk 5]
```diff
@@ -1,23 +1,19 @@
-/* eslint-disable */
+/*eslint-disable*/
```
Comment (arslanbisharat) [inline src/index.js:1]: - [ ] Let's not disable linter rules.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @Abel-desu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Abel-desu 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**